pomegranate/base.pyx

```diff
@@ -41,16 +35,22 @@ cdef class Model( object ):
 	def set_params(self, state):
 		self.__setstate__(state)
 
-	def copy( self ):
+	def to_json(self):
+		"""Serialize this object into JSON format."""
+
+		raise NotImplementedError
+
+	@classmethod
+	def from_json( cls, json ):
+		"""Deserialize this object from its JSON representation."""
+		raise NotImplementedError
+
+	def copy(self):
 		"""Return a deep copy of this distribution object.
 
 		This object will not be tied to any other distribution or connected
 		in any form.
 
-		Paramters
-		---------
```

Comment:
Why are we getting rid of this? I like having this, in order to specific explicitly that there are no parameters to pass in.